config/routes.rb

```diff
@@ -154,11 +154,15 @@
     resources :reads, only: [:create]
   end
 
+  namespace :webhooks do
+    post "/mailchimp/:secret/unsubscribe", to: "mailchimp_unsubscribes#create", as: :mailchimp_unsubscribe
```

Comment:
We can probably change this at one point and use routing DSL trickery, but for now this seems good enough.